src/Display/Tabs/NavigationTabs.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+
+import classNames from 'classnames';
+
+import TabPane, { Props as TabPaneProps } from './TabPane';
+
+import {
+  TabsContainer,
+  TabsHeader,
+  TabsBody,
+} from '../../Style/Display/TabsStyle';
+
+const NavigationTabs: Tabs = ({activeTab, onTabClick, children, variant, className, alignment}) => {
+  const [currentIndex, setCurrentIndex] = React.useState(0);
+  const activeTabOrIndex: string | number = activeTab || currentIndex;
+  const childrenArray = React.Children.toArray(children);
+  const currentChild = childrenArray[currentIndex];
+
+  const handleTabClick = (index: number, tab: React.ReactText) => {
+    const listener = () => {
+      setCurrentIndex(index);
+      if (onTabClick) {
+        onTabClick(tab);
+      }
+    };
+    return listener;
+  };
+
+  return (
+    <TabsContainer className={classNames(`aries-tabs-${alignment}`, className)}>
+      <TabsHeader className="tabs-header">
+        <ul
+          className={classNames(`${alignment}-tabs-list`, `${variant}`)}
+          role="tablist">
+          {React.Children.map(children, (data: React.ReactElement<TabPaneProps>, index) => {
+            const tabLabel = data.props.label || index;
+            return(
+            <li
+              className={
+                classNames(
+                  `tab-${tabLabel}`,
+                  { 'active': activeTabOrIndex === tabLabel },
```

Comment:
Oh now that there is no need for `kebab-case`, this can just be `active` without the quotes

Reply:
Ah forgot to remove the quotes.
Done